index.mjs

```diff
@@ -0,0 +1,88 @@
+// Book class constructor
```

Comment:
- [ ] I was unable to navigate through your app because of this file name. When changed it to `index.js` it worked. Kindly fix that.

Reply:
@TedLivist I had to change it because I was getting I Webhint error related to node and how does it process modules, I can change it back but please take this into consideration👀

Reply:
@TedLivist Already fix it, the file path needed to be corrected in the `<script>` tag in the index.html file. Thanks for pointing this out